README.md

```diff
@@ -40,7 +40,33 @@ npm install plotly.js
 <script type="text/javascript" src="https://cdn.plot.ly/plotly-latest.min.js"></script>
 ```
 
-Read the [Getting started page](https://plot.ly/javascript/getting-started/) for examples.
+#### Webpack Usage
+
+Plotly.js uses a browserify transform (glslify) to transform shaders. To make this work with Webpack, you will need to install [ify-loader]() and add it to your `webpack.config.json`.
+
+Read the [Getting started page](https://plot.ly/javascript/getting-started/) for more examples.
```

Comment:
I'd put the *Webpack Usage* section below the **Modules** sections because a simple `require('plotly.js')` with webpack will grab the `dist/` version so no need to worry about `ify-loader` in the simple case.